app/src/main/java/com/dgaf/happyhour/Controller/DealListFragment.java

```diff
@@ -70,6 +80,18 @@ public void onActivityCreated(Bundle savedInstanceState) {
 
         mRecyclerView.addItemDecoration(new DealListDecoration(getActivity()));
 
+        onRefresh();
+
+    }
+
```

Comment:
This seems odd.  Both methods need to be overridden, but we want an identical behavior?